.github/workflows/email-add-entry.yml

```
@@ -134,13 +134,16 @@ jobs:
 
           gh pr create \
             --title "Add $CITATION_KEY" \
-            --body "Added from email
+            --body "$(cat <<'PREOF'
+Added from email
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)